Feature/vdx 224/energy shr 1

zoemaas opened the pull request.

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Implemented basic changes of the design;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Implemented HighlightBox;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Implemented caption above the content title;

sanderPostma committed:
Make tcp port configurable

sanderPostma committed:
Make tcp port configurable

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Latest changes;

Commit:
Merge remote-tracking branch 'origin/feature/VDX-224/energy-shr-1' into feature/VDX-224/energy-shr-1

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Added placeholder file;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Updated fonts;
   - Updated the menu logo size;
   - Fixed partners spacing;
   - Fixed carousel opacity/color;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Updated caret color;
   - Updated AssetType color;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Increased distance between partners and links;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Changed carousel image;
   - Updated font weight;
   - Added footer top border;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Fixed font weight;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Refactored font-weight;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Refactored font-weight;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Updated catalogue;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Updated catalogue;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Fixed publish page;

BtencateSphereon commented on src/components/Home/Header.module.css:
```diff
@@ -22,7 +22,12 @@
+
+.title {
+  font-weight: var(--font-weight-bolder);
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.14);
```
When i load up the portal i can see there is a box shadow around: Solving mobility challenges.

is this ment to be there? it looks kinda weird

zoemaas replied:
Removed!

BtencateSphereon commented on src/components/Publish/index.tsx:
```diff
@@ -279,10 +280,15 @@ export default function PublishPage({
+            />
+          </div>
+          <div style={{ marginTop: '2rem', marginBottom: '2rem' }}>
+            <Wallet />
```
the button says CONNECTWALLET, should that not be CONNECT WALLET? not sure where the issue comes from

zoemaas replied:
It comes from the original code. The text is correct, the display is wrong. Fixed.

BtencateSphereon commented on content/pages/home/content.json:
```diff
@@ -1,23 +1,39 @@
   "firstTimeVisiting": {
     "title": "First time visiting",
-    "text": "Before interacting with all the marketplace functionalities, you'll have to ensure you have the correct setup (wallet, testnet, tokens). We can help you jump-start the process!",
+    "text": "Lorem ipsum dolor sit amet consectetur. Non vel risus integer aliquam. Parturient ac vulputate.",
```
we need the correct text here?

zoemaas replied:
We don't have a correct text yet. The text used is from FMDM porta. The last update from Maria is that she received the text from the client but she needs to clean it up.

BtencateSphereon replied:
should that not be implemented in this ticket? if not, lets create tickets for that

zoemaas replied:
[VDX-239](https://sphereon.atlassian.net/browse/VDX-239) created to handle it.

BtencateSphereon commented on content/pages/home/content.json:
```diff
@@ -1,23 +1,39 @@
+  },
+  "getInvolved": {
+    "title": "Get involved",
+    "text": "Lorem ipsum dolor sit amet consectetur. Non vel risus integer aliquam. Parturient ac vulputate.",
```
we need the correct text here?

zoemaas replied:
We don't have a correct text yet. The text used is from FMDM portal. The last update from Maria is that she received the text from the client but she needs to clean it up.

BtencateSphereon replied:
should that not be implemented in this ticket?  if not, lets create tickets for that

zoemaas replied:
Will be handled in this ticket: [VDX-239](https://sphereon.atlassian.net/browse/VDX-239)

BtencateSphereon commented on src/components/@shared/AssetTeaser/index.module.css:
```diff
@@ -23,6 +23,7 @@
 a.link:hover,
 a.link:focus {
   box-shadow: var(--teaser-hover-box-shadow);
+  color: rgba(147, 147, 147); /* What should happen with the text color? */
```
should this note be addressed?

zoemaas replied:
Removed!

BtencateSphereon commented on src/components/@shared/Markdown/index.tsx:
```diff
@@ -17,14 +19,28 @@ const Markdown = ({
+    <div className={mergedClassName}>
+      <div
+        dangerouslySetInnerHTML={{ __html: content }}
+        /*
```
this can be removed? and should we not bring back the other note?

zoemaas replied:
Removed!

BtencateSphereon replied:
What about the other note?

zoemaas replied:
Fixed!

BtencateSphereon commented on src/components/@shared/Markdown/index.tsx:
```diff
@@ -17,14 +19,28 @@ const Markdown = ({
+      {subContent && (
+        <div
+          style={{ fontWeight: 'var(--font-weight-base)' }}
+          className={styles.subtext} // add a class for subtext styling
```
what about the note?

zoemaas replied:
Inherited comment. Removed!

BtencateSphereon commented on content/pages/home/header.json:
```diff
@@ -1,16 +1,14 @@
-    "/images/headerCarousel/2-bikes.jpeg",
-    "/images/headerCarousel/3-ev-charge.jpeg",
-    "/images/headerCarousel/4-e-scooter.jpeg"
+    "/images/headerCarousel/1-Energy-SHR.png",
```
can't find the place in the code, but the carousel looks very washed. if you inspect it and disable

--image-height: 50rem, the colors looks exactly like the design. not sure what the cause is

zoemaas replied:
In the last meeting it was decided to have the carousel working but repeating the same picture. Fixed the color issue

BtencateSphereon commented on content/pages/home/header.json:
```diff
@@ -1,16 +1,14 @@
 {
   "body": "Find or publish datasets and AI algorithms for solving challenges in mobility. Experience and leverage decentralized technologies and the European Gaia-X data sharing concepts. Facilitating public-private, cross-sector, cross-border collaboration while maintaining digital sovereignty.",
   "cta": [
+    { "label": "Learn more", "link": "#" },
```
this link is not working, should it go anywhere?

zoemaas replied:
At the moment there is no design/page or decision to where they should go

BtencateSphereon replied:
So what should be the next step for this? having buttons that don't do anything without a next action to take is not very ideal

zoemaas replied:
Maria will have a meeting with Maarten today and possibly with Niels to define it. If they don't come up with a solution we can always create a ticket to fix it. This ticket was messy from the beginning.

zoemaas replied:
"Learn more" link was removed from the design. So I removed it from the code as well. I think it's fixed now.

Commit:
VDX-233 Implement faceted search to the Energy SHR portal

   - Removed box shadow from header .title

Commit:
VDX-233 Implement faceted search to the Energy SHR portal

   - Fixed the connect wallet button text;

Commit:
VDX-233 Implement faceted search to the Energy SHR portal

   - Removed obsolete comments;

Commit:
VDX-233 Implement faceted search to the Energy SHR portal

   - Fixed carousel image color;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Fixed link to the onboarding page;
   - Brought back the original comment in the Markdown component;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Fixed on focus from the Learn more link;

Commit:
VDX-224 Integrate Energy SHR design into Portal copy of FMDM

   - Removed "Learn more" link;

BtencateSphereon reviewed (approved):
LGTM
